vars/piperExecuteBin.groovy

```diff
@@ -8,9 +8,14 @@ import com.sap.piper.Utils
 
 import static com.sap.piper.Prerequisites.checkScript
 
-void call(Map parameters = [:], stepName, metadataFile, List credentialInfo, failOnMissingReports = false, failOnMissingLinks = false) {
+void call(Map parameters = [:], stepName, metadataFile, List credentialInfo, failOnError = false, failOnMissingReports = false, failOnMissingLinks = false) {
 
-    handlePipelineStepErrors(stepName: stepName, stepParameters: parameters) {
+    handlePipelineStepErrorsParameters = [stepName: stepName, stepParameters: parameters]
+    if (failOnError) {
+        handlePipelineStepErrorsParameters.failOnError = true
```

Comment:
I can't see where `failOnError` is used.
Wouldn't it be better to simply say 
```
handlePipelineStepErrorsParameters.failOnError = failOnError
``` ? In this case that property is always set. Otherwise we have it set to `true`, but never set explicitly in case it is false. Works also like that with groovy, but having it set is more explicit.

Reply:
we do not want to set it hard here since we need to rely on the user config ...